src/pages/my-lessons/MyLessons.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+import Box from '@mui/material/Box'
+
+import Tab from '~/components/tab/Tab'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+import { styles } from '~/pages/my-lessons/MyLessons.styles'
+
+const listDataMock = [
```

Comment:
move this to MyLessons.constants file

Reply:
It will be deleted later, so maybe we should leave it as it is ?

Reply:
we move all constants or mock to constants file, it will be better

Reply:
done